Sources/SwiftIfConfig/SwiftIfConfig.docc/SwiftIfConfig.md

```diff
@@ -0,0 +1,36 @@
+# `SwiftIfConfig`
```

Comment:
Looks like titles in docc can have code font. 

https://swiftpackageindex.com/swiftlang/swift-syntax/main/documentation/swiftifconfig/swiftifconfig

Reply:
*can't. Will fix